CHANGELOG.md

```diff
@@ -8,13 +8,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.0.5] - ???
 
 ### Added
+
 - Added `VueObserveVisibility` and `VueLazyload` dependency
 
+### Changed / Improved
+
+- Updated o-personal-details to match latest designs(#601)
```

Comment:
I think it'd be good if we mentioned what exactly updated

Reply:
Ok